tests/api/v1/events/get.js

```diff
@@ -33,8 +33,10 @@ describe('tests/api/v1/events/get.js >', () => {
   let testEvent = u.getStandard();
   let testEventOutput;
   let testEvent2 = u.getStandard();
+  testEvent2.context.type = 'Comment';
```

Comment:
Is the type value case-sensitive? (I think it should be case-INsensitive, unless you have a compelling reason not to do so. Add tests for case-insensitivity.)